ddtrace/span.py

```diff
@@ -297,8 +300,8 @@ def set_tag(self, key, value=None):
             log.warning("error setting tag %s, ignoring it", key, exc_info=True)
 
     def _set_str_tag(self, key, value):
-        # type: (str, str) -> None
-        self.meta[key] = stringify(value)
+        # type: (Text, AnyStr) -> None
```

Comment:
I'm fine with this but `Text` seems to be slightly incoherent compared with `set_tag` (whose typing is wrong).
The latter checks for `six.string_types` which includes both `str` and `unicode` on Python 2.
Your typing here would only include `unicode` on Python 2.

Reply:
Fixed